### ISSUE TITLE
run on machines with only later versions of .NET Core than 2.x

### ISSUE THREAD

**Issue by adamralph** [adamralph/minver#244]
Currently:

```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
  - The following frameworks were found:
      3.0.0-preview7-27912-14 at [C:\hostedtoolcache\windows\dotnet\shared\Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The .NET Core frameworks can be found at:
  - https://aka.ms/dotnet-download
```

**Comment by adamralph** [comment]
On hold until .NET Core 3 RTM.

**Comment by adamralph** [comment]
https://github.com/adamralph/minver/issues/112 would probably allow this.

**Comment by adamralph** [comment]
Released in [2.0.0-alpha.2](https://www.nuget.org/packages/MinVer/2.0.0-alpha.2).
